great_expectations/validator/metrics_calculator.py

```diff
@@ -140,32 +140,28 @@ def get_metrics(
     def compute_metrics(
         self,
         metric_configurations: List[MetricConfiguration],
+        runtime_configuration: Optional[dict] = None,
```

Comment:
I'm always wary of updating public signatures - why is this necessary?

Reply:
@cdkini Because it was an omission before -- the methods it calls require this, and we should uphold it.  The optional aspect should make any public usage seamless.  Thanks.